zingo-sync/src/primitives.rs

```diff
@@ -284,7 +290,7 @@ pub struct WalletNote<N, Nf: Copy> {
     nullifier: Option<Nf>, //TODO: syncing without nullfiier deriving key
     /// Commitment tree leaf position
```

Comment:
Position is none for mempool and some for confirmed, is this the only source of truth for whether a walletnote is confirmed? If so this needs to be clearer, a comment will probably do the trick,,
```suggestion
    /// Commitment tree leaf position if confirmed, or None for an unconfirmed note
```

Reply:
well... the single source of truth for whether a note is confirmed is whether the transaction it belongs to is confirmed. I imagine the position will be None for notes in confirmed transactions when we add support for viewing key only wallets for extra speed up right?